examples/c/minimal_legacy.bpf.c

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
+#define BPF_NO_GLOBAL_DATA
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+// Create an array with 1 entry instead of a global variable
+// which does not work with older kernels
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__uint(max_entries, 1);
+	__type(key, u32);
+	__type(value, pid_t);
+} my_pid SEC(".maps");
```

Comment:
maybe call this `my_pid_map` so you can have `my_pid` local variable below? `user_pid` is a bit confusing